src/lib/Zikula/Bundle/CoreInstallerBundle/Controller/AjaxInstallController.php

```diff
@@ -337,8 +337,8 @@ private function decodeParameters($params)
     public function reloadRoutes()
     {
         // fire MODULE_INSTALL event to reload all routes
-        $event = new ModuleStateEvent($this->container->get('kernel')->getModule('ZikulaRoutesModule'));
-        $this->container->get('event_dispatcher')->dispatch(CoreEvents::MODULE_POSTINSTALL, $event);
+//        $event = new ModuleStateEvent($this->container->get('kernel')->getModule('ZikulaRoutesModule'));
+//        $this->container->get('event_dispatcher')->dispatch(CoreEvents::MODULE_POSTINSTALL, $event);
```

Comment:
basically yes. I disabled it because it isn't needed. I left the code commented because removing it means the ajax stage should be fully removed and I didn't want to take the time to do that yet.